indico/modules/users/templates/personal_data.html

```diff
@@ -31,6 +31,9 @@
                                 {% trans %}Unblock User{% endtrans %}
                             </button>
                         {% endif %}
+                        {% if delete_user_from_ui_enabled %}
+                            <ind-user-delete-button user-id={{ (user.id if user != session.user else none) | tojson }} user-is-admin="{{ user.is_admin | tojson }}" user-first-name="{{ user.first_name }}" user-last-name="{{ user.last_name }}"></ind-user-delete-button>
```

Comment:
I'm wondering if this might be better, since it does make it tricky to read when all the attributes are on one line.

```suggestion
<ind-user-delete-button 
   user-id={{ (user.id if user != session.user else none) | tojson }}
   user-is-admin="{{ user.is_admin | tojson }}"
   user-first-name="{{ user.first_name }}"
   user-last-name="{{ user.last_name }}">
</ind-user-delete-button>
```

Reply:
okay formatted